src/System.Xml.Serialization/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,4 @@
+DEFAULT_APPNAME Error: 0 : Unable to resolve assembly 'Assembly(Name=mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)' referenced by the contract assembly 'Assembly(Name=System.Xml.Serialization, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)'.
```

Comment:
It is trying to find these in the "contractsDepends" directories. One of those will be the directory of the contract but that isn't where I would expect those to be resolved from. We should look further in the logic to figure out how we correctly add a contractDepends directory to the apicompat tool so that it will find them wherever the build found them.